### PR TITLE
Improve trigger condition in github actions

### PR DESCRIPTION
**Pull request by idubnori**
## **User description**
fix: #827

___

## **Type**
enhancement, documentation

___

## **Description**
- Added logging in the GitHub Action runner to provide information when an action is skipped and the reason for it.
- Updated the GitHub Action configuration documentation to include `types` for `pull_request` events. This specifies the action to run on `opened`, `reopened`, `ready_for_review`, and `review_requested` events, improving the trigger condition.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>github_action_runner.py</strong><dd><code>Add Logging for Skipped Actions</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
pr_agent/servers/github_action_runner.py

- Added logging for skipping actions with a specific reason.

</details>
    

  </td>
  <td><a href="https://github.com/Codium-ai/pr-agent/pull/828/files#diff-15f9c11da031449998128bdbba057d768853e836eedf44563be71b3847be80e0">+2/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>github.md</strong><dd><code>Specify GitHub Action Trigger Conditions</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
docs/docs/installation/github.md

<li>Added <code>types</code> to <code>pull_request</code> event to specify when the GitHub Action <br>should run.<br> <li> This change specifies the action to run on <code>opened</code>, <code>reopened</code>, <br><code>ready_for_review</code>, and <code>review_requested</code> events.<br>

</details>
    

  </td>
  <td><a href="https://github.com/Codium-ai/pr-agent/pull/828/files#diff-e73a6a650491a94e59d2b22db828e80548158694e07b521342f7dfe6fa3f1866">+2/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: docs/docs/installation/github.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ You can use our pre-built Github Action Docker image to run PR-Agent as a Github
 ```yaml
 on:
   pull_request:
+    types: [opened,reopened,ready_for_review,review_requested]
   issue_comment:
 jobs:
   pr_agent_job:
```
Comment (mrT23) [inline docs/docs/installation/github.md:13]: /ask can i limit the 'issue_comment' to ignore bots ?  can I limit it to ignore message not starting with '/' ?

**Comment by mrT23** [comment]
@idubnori 
This PR is reasonable.
Remove the 'review_requested' flag, as it is a bug we already fixed in github app

Also, another optimization I want is to ignore bots. So the action should be:

```
on:
  pull_request:
    types: [opened, reopened, ready_for_review]
  issue_comment:

jobs:
  pr_agent_job:
    if: ${{ github.event.sender.type != 'Bot' }}
```

there are additional optimizations that can be done (ignore comments if they don't start with '/', for example), but lets leave them to future PRs.

Adpat the PR to the condition above, and we can merge it

**Comment by idubnori** [comment]
@mrT23 Thank you for your reviewing. fixed them.

**Review by mrT23 — APPROVED** [review]

**Review by barnett-yuxiang — COMMENTED** [review]
cool

**Pull request merged**